core/common/src/IModelError.ts

```diff
@@ -32,34 +32,41 @@ export class IModelError extends BentleyError {
   }
 }
 
+/** The state of a lock. See [Acquiring locks on elements.]($docs/learning/backend/ConcurrencyControl.md#acquiring-locks-on-elements).
+ * @public
+ */
+export enum LockState {
+  /** The element is not locked */
+  None = 0,
+  /** Holding a shared lock on an element blocks other users from acquiring the Exclusive lock it. More than one user may acquire the shared lock. */
+  Shared = 1,
+  /** A Lock that permits modifications to an element and blocks other users from making modifications to it.
+   * Holding an exclusive lock on an "owner" (a model or a parent element), implicitly exclusively locks all its members.
+   */
+  Exclusive = 2,
+}
+
 /** Detailed information about a particular object Lock that is causing the Lock update conflict.
+ * An example of a lock update conflict would be attempting to use [LockControl.acquireLocks]($backend) on an object that is already locked by another Briefcase.
  * @public
 */
 export interface ConflictingLock {
   /** Id of the object that is causing conflict. */
   objectId: string;
   /**
-     * The level of conflicting lock. Possible values are {@link LockLevel.Shared}, {@link LockLevel.Exclusive}.
-     * See {@link LockLevel}.
+     * The level of conflicting lock. Possible values are {@link LockState.Shared}, {@link LockState.Exclusive}.
+     * See {@link LockState}.
      */
-  lockLevel: LockLevel;
+  lockLevel: LockState;
```

Comment:
```suggestion
  state: LockState;
```

Nitpick but "level" doesn't make sense any more and "lock" is redundant.